### ISSUE TITLE
[CoE Starter Kit - QUESTION] Admin | Sync Template Flows not running after environment action (restore / copy)

### ISSUE THREAD

**Issue by dtegg** [microsoft/coe-starter-kit#5007]
### What is your question?

We've been using the CoE Starter Kit within our company for a couple of years, but have recently looked to follow guidance to set up a [dedicated testing environment](https://learn.microsoft.com/en-us/power-platform/guidance/coe/after-setup?source=recommendations#testing-strategy) to test updates.

This new Sandbox environment was created 11/23/2022 & we installed CoE Starter Kit v4.2 in December 2022.
Last week 3/9 we installed v4.3 without any errors and the main Admin | Sync Template v3 Flow has run successfully for a number of days, but doesn't appear to be triggering the other Admin | Sync Template Flows, in particular Apps & Flows.
When we visit the Power Platform Admin View App, we noticed that there are no Flows or Apps showing after 11/23/2022.

Appreciate any support you can provide.  
![image](https://user-images.githubusercontent.com/64900433/225493268-a3ca53b5-cd30-4dba-9498-8731041bcddf.png)
![image](https://user-images.githubusercontent.com/64900433/225493364-64047e52-99e9-4a35-bcab-70d230f9f52e.png)

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

4.3

### What app or flow are you having the issue with?

Most of the Admin | Sync Template v3 Flows

**Comment by Jenefer-Monroe** [comment]
Hello, I have a theory about the cause of this as we have seen it locally before as well. 
Can you please check for me by going to the products admin page: https://admin.powerplatform.microsoft.com/ > Environments > Find your CoE environment > Click History from the ribbon
![image](https://user-images.githubusercontent.com/89584709/217530536-ff6a67a6-1af8-4092-8c98-eb2c2875041c.png)

Please share what you see in the history.
![image](https://user-images.githubusercontent.com/89584709/217530654-9c0e1d44-bc5f-4e29-a6fd-d9df515d8a7b.png)

**Comment by dtegg** [comment]
Hi Jenefer,
The environment was created 11/23/2022, which aligns with the latest Flows/Apps we're seeing in the Power Platform Admin View (i.e. nothing showing that was created after 11/23/2022)
![image](https://user-images.githubusercontent.com/64900433/225736151-0b929687-f38d-481d-ae57-1afd7c3e6112.png)

![image](https://user-images.githubusercontent.com/64900433/225736173-25ef6bbe-4504-4f24-891d-2be4489b838d.png)

Thanks

**Comment by manuelap-msft** [comment]
Hello,

we've raised this internally as well and got the following response
1. After environment copy and restore, the environment is put in [administration mode](https://learn.microsoft.com/en-us/power-platform/admin/admin-mode)
2. Dataverse triggers do not trigger for flows in administration mode
3. You'll have to disable administration mode on the environment, see: https://learn.microsoft.com/en-us/power-platform/admin/admin-mode#set-administration-mode
4. After taking the environment out of admin mode, flows may not trigger as expected until caches have been cleared; this can take up to 24 hours.

**Comment by dtegg** [comment]
Thank you both for your responses.
That makes sense, as we copied the Prod Starter Kit Env, to use a starting point for the Sandbox.
It was in Admin mode & after turning this off, the Admin Sync Flows are running again.

We are however noticing that the Sandbox PBI Report is showing different counts for Flows than Prod (almost like the Flows created in the last few months haven't All been added).  We've set the Starter Kit Env Variable "FullInventory" to yes & it looks as though App and Environment counts match Prod after running through for a day or so.  Is there anything else we can check here? Or should we just let it run for another couple of days to iron out by itself?
